packages/@react-native-windows/cli/src/runWindows/utils/autolink.ts

```diff
@@ -401,6 +411,48 @@ async function updateAutoLink(
         changesNecessary;
     }
 
+    // Generating props for app project consumption
+    let propertiesForProps = '';
+    let addCsModuleSupport: boolean = false;
+
+    if (projectLang === 'cpp') {
+      for (const dependencyName in windowsDependencies) {
+        windowsDependencies[dependencyName].projects.forEach(project => {
+          if (project.directDependency && project.projectLang === 'cs') {
+            addCsModuleSupport = true;
+          }
+        });
+      }
+
+      if (addCsModuleSupport) {
+        propertiesForProps += `\n    <ConsumeCSharpModules>true</ConsumeCSharpModules>`;
+      }
```

Comment:
```suggestion
    let csModule = '';

    if (projectLang === 'cpp') {
      for (const dependencyName in windowsDependencies) {
        windowsDependencies[dependencyName].projects.forEach(project => {
          if (project.directDependency && project.projectLang === 'cs') {
            csModule = project.projectName;
          }
        });
      }

      const addCsModuleSupport: boolean = (csModule !== '');

      if (addCsModuleSupport) {
        propertiesForProps += `\n    <!-- Auto-generated by autolinking due to dependency on C# module ${csModule}: -->`;
        propertiesForProps += `\n    <ConsumeCSharpModules>true</ConsumeCSharpModules>`;
      }
```

Reply:
I like this idea, but more than one project can be responsible, so I'll build an array.